src/main/kotlin/org/taktik/freehealth/middleware/service/impl/TarificationServiceImpl.kt

```diff
@@ -331,14 +331,14 @@ class TarificationServiceImpl(private val stsService: STSService) : Tarification
                         base = "/${nodeDescr(node.parentNode)}$base"
                         node = node.parentNode
                     }
-                    val elements =
+                    var elements =
```

Comment:
Nope... Pas de var

Reply:
écris plutôt: val elements = if(elements.isNotEmpty()) ... else ....

Reply:
Je vais procéder comme suit en utilisant uniquement des val puisque que je ne peux pas utiliser elements pour l'assigner lui-même:

`val initialElements = ConsultTarifErrors.values.filter {
                        it.path == base && it.code == ec && (it.regex == null || url.matches(Regex(".*${it.regex}.*")))
                    }

                    val elements = if (initialElements.isEmpty()) {
                        // IOs sometimes are overeager to provide us with precise xpath. Let's try again while truncating after the item
                        val truncatedBase = base.replace(Regex("(.+/item.+?)/.*"), "$1")
                        ConsultTarifErrors.values.filter {
                            it.path == truncatedBase && it.code == ec && (it.regex == null || url.matches(Regex(".*${it.regex}.*")))
                        }
                    } else {
                        initialElements
                    }`

Reply:
@aduchate J'ai effectué le changement